sycl/doc/extensions/deprecated/sycl_ext_oneapi_extended_atomics.asciidoc

```diff
@@ -27,7 +27,7 @@ NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are tradema
 
 NOTE: This document is better viewed when rendered as html with asciidoctor.  GitHub does not render image icons.
 
-This document describes an extension that introduces the `cl::sycl::intel::atomic_ref` class, which exposes additional functionality aligned with the +std::atomic_ref+ class from {cpp}20.
```

Comment:
We should probably keep references to SYCL 1.2.1 unchanged, it seems cl::sycl is accurate in these cases.

Reply:
I wasn't sure how to proceed with those actually. The code that we have in-tree wouldn't have "cl::", so having docs reference it isn't good either.

Reply:
In other words, are those docs "abstract" and (from the technical perspective) should have lived in the khronos repo, or are they the reflection of our actual implementation?

Reply:
All docs changes are generally good I think! In supported extensions we should use sycl:: and that will match implementation. I'm just taking about a few very specific references (in deprecated extensions docs) to SYCL 1.2.1 standard where namespaces are cl::sycl in fact. I had to be more specific in my references.

Reply:
I agree with this.  It would be good to retain cl:: references in specific instances when referring to 1.2.1 specs.

Reply:
I had a chat with Pavel yesterday, decided to revert the docs changes under "deprecated/" folder completely. I had to re-focus on another regression, will upload the change once I'll return to this work.